biojava-structure/src/main/java/org/biojava/nbio/structure/asa/AsaCalculator.java

```diff
@@ -21,12 +21,13 @@
 package org.biojava.nbio.structure.asa;
 
 import org.biojava.nbio.structure.*;
+import org.biojava.nbio.structure.contact.Contact;
+import org.biojava.nbio.structure.contact.Grid;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import javax.vecmath.Point3d;
-import java.util.ArrayList;
-import java.util.TreeMap;
+import java.util.*;
```

Comment:
Minor point, but maybe we should try to avoid wildcard imports?

Reply:
That's an IntelliJ feature. I do like individual imports better too... I'll have to check if I can configure IntelliJ to do it.